web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
```

Comment:
feel like this is something that would be relatively easy to clean up later. How about  we come back and clean this up once we're farther along the tournament API? Should be relatively straightforward if we want to replace with tournament IDs.

Personally, I like having more full players, because it means that TDs can run a tournament offline and submit the results via the web form. In that case, the submission process can auto-register users if they don't exist.